extract/dcinside/driver_test.py

```
@@ -54,20 +55,23 @@ def lambda_handler(event, context):
-#         print("Test Successfully Ended")
-#         driver.quit()
-#     else:
-#         print("Something Wrong in Code")
\ No newline at end of file
+if __name__=="__main__":
+    """로컬 및 도커 컨테이너에서 테스트 할 용도"""
+    print("Starting Test ...")
+    try:
+        driver = get_driver()
+        print("Chrome driver has set.")
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)